### PR TITLE
Add links to AHB eligible RHEL and SLES offers

### PR DESCRIPTION
**Pull request by MarckK**

### CODE REVIEW

**Comment by MarckK** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Review by vvarshney06 — APPROVED** [review]

**Comment by vvarshney06** [comment]
#sign-off

**Comment by MarckK** [comment]
Hi @fossygirl! Could I have some help with this blocked PR? 
Thank you!

**Comment by MarckK** [comment]
Hi @Court72! Thank you for your comments. I don't believe that I have access to the private repo and therefore can't move the commits. 

I will need the PR review team's assistance.

**Comment by Jak-MS** [comment]
@MarckK - we will start this process as we have time, probably tomorrow and post updates here in this PR.

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 1]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
+Azure Hybrid Benefit (AHB) for Linux lets you easily switch the software subscription model for your VM, enabling flexibility and savings.
+The conversion between subscription models is accomplished without a redeployment of your VM, avoiding any downtime risk.
 
-Customers may see combined savings estimated to up to 76% with Azure Hybrid Benefit for Linux and three-year Azure Reserved VM Instances. Savings estimates are based on one standard D2s v5 Azure VM with RHEL or SLES subscription in the East US region running at a pay-as-you-go rate vs a reduced rate for a three-year reserved instance plan. Based on Azure pricing as of September 2024. Prices subject to change. Actual savings may vary based on location, instance type, or usage.
+There are two subscription models on Azure which you can switch seemlessly between using Azure Hybrid Benefit.
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:22]:
```suggestion
There are two subscription models on Azure which you can switch seamlessly between using Azure Hybrid Benefit. 
```

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 2]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
+The conversion between subscription models is accomplished without a redeployment of your VM, avoiding any downtime risk.
 
-Customers may see combined savings estimated to up to 76% with Azure Hybrid Benefit for Linux and three-year Azure Reserved VM Instances. Savings estimates are based on one standard D2s v5 Azure VM with RHEL or SLES subscription in the East US region running at a pay-as-you-go rate vs a reduced rate for a three-year reserved instance plan. Based on Azure pricing as of September 2024. Prices subject to change. Actual savings may vary based on location, instance type, or usage.
+There are two subscription models on Azure which you can switch seemlessly between using Azure Hybrid Benefit. 
+* The first enables you to bring your own REHL or SLES subscription directly to Azure.
+With this Bring-Your-Own-Subscription (BYOS) model, you only pay Azure for the infrastructure costs of your VM as the software fee is covered by your RHEL or SLES subscription.
+
+* The second Pay-As-You-Go (PAYG) model enables you to pay for RHEL and SLES subscriptions as you use them.
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:26]:
```suggestion
* The second pay-as-you-go model enables you to pay for RHEL and SLES subscriptions as you use them. 
```

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 3]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
+Based on Azure pricing as of September 2024. Prices subject to change. Actual savings may vary based on location, instance type, or usage.
 
 > [!TIP]
 > Try the **[Azure Hybrid Benefit Savings Calculator](https://azure.microsoft.com/pricing/hybrid-benefit/#calculator)** to visualize the cost saving benefits of this feature.
 
-## Defining Pay-as-you-go (PAYG) and Bring-your-own-subscription (BYOS)
+## Defining Pay-As-You-Go (PAYG) and Bring-Your-Own-Subscription (BYOS)
 
-In Azure, there are two main licensing pricing options: 'pay-as-you-go' (PAYG) and 'bring-your-own-subscription' (BYOS). 'PAYG' is a pricing option where you pay for the resources you use on an hourly or monthly basis. You only pay for what you use and can scale up or down as needed. On the other hand, 'BYOS' is a licensing option where you can use your existing licenses for certain software, in this case RHEL and SLES, on Azure virtual machines. You can use your existing licenses and don't have to purchase new ones for use in Azure.
+In Azure, there are two main licensing pricing options: Pay-As-You-Go (PAYG) and Bring-Your-Own-Subscription (BYOS).
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:41]:
```suggestion
In Azure, there are two main licensing pricing options: Pay-as-you-go and Bring-Your-Own-Subscription (BYOS). 
```

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 4]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
 
 > [!TIP]
 > Try the **[Azure Hybrid Benefit Savings Calculator](https://azure.microsoft.com/pricing/hybrid-benefit/#calculator)** to visualize the cost saving benefits of this feature.
 
-## Defining Pay-as-you-go (PAYG) and Bring-your-own-subscription (BYOS)
+## Defining Pay-As-You-Go (PAYG) and Bring-Your-Own-Subscription (BYOS)
 
-In Azure, there are two main licensing pricing options: 'pay-as-you-go' (PAYG) and 'bring-your-own-subscription' (BYOS). 'PAYG' is a pricing option where you pay for the resources you use on an hourly or monthly basis. You only pay for what you use and can scale up or down as needed. On the other hand, 'BYOS' is a licensing option where you can use your existing licenses for certain software, in this case RHEL and SLES, on Azure virtual machines. You can use your existing licenses and don't have to purchase new ones for use in Azure.
+In Azure, there are two main licensing pricing options: Pay-As-You-Go (PAYG) and Bring-Your-Own-Subscription (BYOS). 
+With PAYG you only pay for the resources you use and can scale up or down as needed.
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:42]:
```suggestion
With pay-as-you-go models you only pay for the resources you use and can scale up or down as needed. 
```

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 5]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
-You can use Azure Hybrid Benefit to switch back to pay-as-you-go billing at any time.
+You can use Azure Hybrid Benefit to switch between the two license options at any time.
+
+## Which Linux Virtual Machines are Eligible for Azure Hybrid Benefit 
+### PAYG 
+The following PAYG RHEL and SLES Marketplace offers are AHB eligible:
+
+##### [RHEL PAYG](#tab/ahbRhelPayg)
+##### RHEL PAYG offers that are ABH eligible:
+**All of the RHEL PAYG eligible offers must have Red Hat, Inc. as the publisher.**
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:58]:
```suggestion
### Pay-as-you-go 
The following pay-as-you-go RHEL and SLES Marketplace offers are AHB eligible:

##### [RHEL pay-as-you-go](#tab/ahbRhelPayg)
##### RHEL pay-as-you-go offers that are ABH eligible:
**All of the RHEL pay-as-you-go eligible offers must have Red Hat, Inc. as the publisher.**
```

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 6]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
+* [Red Hat Enterprise Linux ( RHEL ) for SAP Apps](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-sap-apps?tab=Overview)
+* [RHEL for SAP with HA and Update Services](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-sap-ha?tab=Overview)
+* [Red Hat Enterprise Linux ( RHEL ) with High Availability ( HA ) Add-On](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-ha?tab=Overview)
+* [Red Hat Enterprise Linux ( RHEL ) ARM64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-arm64?tab=Overview)
+* [Red Hat Enterprise Linux Confidential VM](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-cvm?tab=Overview)
+* [Red Hat Enterprise Linux ( RHEL ) RAW](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-raw?tab=Overview)
+
+##### [SLES PAYG](#tab/ahbSlesPayg)
+##### SLES PAYG offers that are ABH eligible:
+**All of the SLES PAYG eligible offers must have SUSE as the publisher.**
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:73]:
```suggestion
##### [SLES pay-as-you-go](#tab/ahbSlesPayg)
##### SLES pay-as-you-go offers that are ABH eligible:
**All of the SLES pay-as-you-go eligible offers must have SUSE as the publisher.**
```

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 7]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
+**All of the RHEL PAYG eligible offers must have Red Hat, Inc. as the publisher.**
+
+The names of the eligible offers are listed below, with links to them in the Azure Marketplace.
+Within these offers, the associated images will be described as 'Pay-As-You-Go'.
+
+* [Red Hat Enterprise Linux ( RHEL )](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-20190605?tab=Overview)
+* [Red Hat Enterprise Linux ( RHEL ) for SAP Apps](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-sap-apps?tab=Overview)
+* [RHEL for SAP with HA and Update Services](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-sap-ha?tab=Overview)
+* [Red Hat Enterprise Linux ( RHEL ) with High Availability ( HA ) Add-On](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-ha?tab=Overview)
+* [Red Hat Enterprise Linux ( RHEL ) ARM64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-arm64?tab=Overview)
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:67]:
```suggestion
* [Red Hat Enterprise Linux ( RHEL ) Arm64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-arm64?tab=Overview)
```

File: articles/virtual-machines/linux/azure-hybrid-benefit-linux.md [hunk 8]
```diff
@@ -14,29 +14,95 @@ ms.reviewer: mattmcinnes
...
+* [Red Hat Enterprise Linux ( RHEL ) RAW](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-raw?tab=Overview)
+
+##### [SLES PAYG](#tab/ahbSlesPayg)
+##### SLES PAYG offers that are ABH eligible:
+**All of the SLES PAYG eligible offers must have SUSE as the publisher.**
+
+The names of the eligible offers are listed below, with links to them in the Azure Marketplace.
+Within these offers, the associated plans and images will be described as a 'Pay as You Go' subscription of SLES.
+* [SUSE Enterprise Linux 15 SP5 + 24x7 Support](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/suse.sles-15-sp5?tab=Overview)
+* [SUSE Enterprise Linux 15 SP5 arm64 + 24x7 Support](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/suse.sles-15-sp5-arm64?tab=Overview)
```
Comment (Court72) [inline articles/virtual-machines/linux/azure-hybrid-benefit-linux.md:78]:
```suggestion
* [SUSE Enterprise Linux 15 SP5 Arm64 + 24x7 Support](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/suse.sles-15-sp5-arm64?tab=Overview)
```

**Comment by Court72** [comment]
These commits were staged in https://github.com/MicrosoftDocs/azure-compute-docs-pr/pull/660 and the tabs rendered as expected. I made some changes to address Acrolinx issues. 

The article **articles/virtual-machines/linux/azure-hybrid-benefit-linux.md** might benefit from an edit pass. An editor will notify the listed author and recent contributors.

**Comment by MarckK** [comment]
Thank you, @Court72! Glad to get this in. 
We will continue editing articles/virtual-machines/linux/azure-hybrid-benefit-linux.md. 👍

**Pull request merged**